src/main/java/com/hollingsworth/arsnouveau/common/entity/pathfinding/MinecoloniesAdvancedPathNavigate.java

```diff
@@ -147,6 +147,10 @@ public boolean isDone() {
 
     @Override
     public void tick() {
+        if (this.mob.isPassenger()) {
```

Comment:
Does vanilla not skip the pathfinder if effectiveAI is false?

Reply:
it does! i forgot to remove it after overwriting isEffectiveAi